pyproject.toml

```diff
@@ -119,6 +119,7 @@ keywords = [
 ]
 packages = [
     { include = "neptune", from = "src" },
+    { include = "neptune/py.typed", from = "src" },
```

Comment:
This line can be safely removed, `include = "neptune"` will pull it in.

Reply:
@SiddhantSadangi can you revert your change?

Reply:
I forgot I had made this change 🙈 
(need to get out of vacation mode)